lib/issue_exporter/import.rb

```diff
@@ -0,0 +1,87 @@
+require 'net/http'
+require 'json'
+
+module IssueExporting
+  class Importer
+    def initialize(files, owner, repo, token)
+      @files = files
+      @owner = owner
+      @repo = repo
+      @token = token
+    end
+
+    def import
+      @files.each do |file|
+        file_contents = read_file file
+        import_json(file_contents) unless file_contents.nil?
+      end
+    end
+
+    private
+
+    def import_json(json_obj)
+      if json_obj.class == Hash
+        import_json_hash json_obj
+      elsif json_obj.class == Array
+        import_json_array json_obj
+      else
+        puts "Unknown object: #{json_obj.class}"
+      end
+    end
+
+    def import_json_array(json_array)
+      json_array.each do |json_hash|
+        import_json_hash json_hash
+      end
+    end
+
+    def import_json_hash(json_hash)
+      create_issue(filter_json json_hash)
+    end
+
+    def filter_json(json_hash)
+      json_hash.select { |k, _v| allowed_properties.include? k }
+    end
+
+    def create_issue(json_obj)
+      uri = IssueExporting.make_uri @owner, @repo, @token
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true if uri.scheme == 'https'
+      request = Net::HTTP::Post.new uri.request_uri
+      request.content_type = 'application/json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.